runtime/lua/vim/treesitter/query.lua

```diff
@@ -249,7 +249,32 @@ local predicate_handlers = {
     end
 
     return false
-  end
+  end,
+
+  ["any-of?"] = (function()
+    local sets = {}
+
+    return function(match, _, source, predicate)
+      local node = match[predicate[2]]
+      local node_text = M.get_node_text(node, source)
+      local start_row, _, end_row, _ = node:range()
```

Comment:
Why are you checking this?

Reply:
I assumed this was necessary for some reason and copied and pasted from the `lua-match?` and `match?` predicate. I think I see now how this would restrict matching of things that span several lines for now good reason.

Reply:
It's because the get_node_text() only gives you the first line. There's an open TODO.

Reply:
I removed the check in the latest commits.